.eslintrc

```diff
@@ -8,7 +8,8 @@
 		"react/destructuring-assignment": "off",
 		"import/prefer-default-export":"off",
 		"react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-		"quotes": [2, "single", { "avoidEscape": true }]
+		"quotes": [2, "single", { "avoidEscape": true }],
+		"no-param-reassign": ["error", { "props": false }]
```

Comment:
Adjusting no-params-reassign was necessary for my helper function, which adds an id parameter to its argument's elements.